setup.cfg

```diff
@@ -0,0 +1,36 @@
+# Note: setup.cfg is normally a config file for Python package publishing. In
+# our case it's just a shared config file for linting and formatting tools
+# because most of them support it automatically.
+
+[flake8]
+# Try to stick to 79, but sometimes being religious *hurts* readability.
+max-line-length = 100
```

Comment:
About the only non-default so far—I am a firm believer in keeping line lengths under control but often feel like the PEP 8 standard of 79 ends up leading to a lot more annoyance and awkward readability than when it's raised just a bit.

Open to your input.